commands-cookbook.md

```diff
@@ -332,9 +334,12 @@ WP_CLI::add_command( 'example hello', $hello_command, array(
 		),
 	),
 	'when' => 'after_wp_load',
+	'longdesc' =>   '## EXAMPLES' . "\n\n" . 'wp example hello Newman',
 ) );
 ```
 
+Note that the longdesc will be appended to the description of the options generated from the synopsis, so is the place to give examples of usage. If there's no synopsis, the longdesc will be used as is.
```

Comment:
Use backticks for `longdesc` key.
Also, grammar in the first sentence at the end seems weird: "so is the place..."

How about:
```
Note that the `longdesc` attribute will be appended to the description of the options generated from the synopsis, so this argument is great for adding examples of usage. If there is no synopsis, the `longdesc` attribute will be used as is to provide a description.
```